BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs

```diff
@@ -24,4 +31,37 @@ public void Index_WhenCalledByAnonymousUser_ReturnsViewWithLoginLink()
         result!.ViewData.Model.Should().BeOfType<AdminViewModel>()
             .Which.CurrentUser.Should().BeNull();
     }
+
+    [Fact]
+    public void GetAdminList_ReturnsViewWith_AdminListModel()
+    {
+        // Arrange
+        var fakeAdminService = A.Fake<IAdminService>();
+        var fakeListOfAdmins = A.Dummy<Task<(List<UserDbModel> CurrentAdmins, List<UserDbModel> PendingAdmins)>>();
+        var fakeAdminController = new AdminController(null!, null!, fakeAdminService);

```

Comment:
Similarly, `AdminControllerTestsBase` already has a fake `AdminController` called `_underTest` (see the test above this one for an example of it being used)